dotnet/CoreLib/Handlers/TextPartitioningOption.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticMemory.Handlers;
+
+/// <summary>
+/// Represents options for text partitioning.
+/// </summary>
+public class TextPartitioningOption
```

Comment:
Maybe rename this to TextPartitioningOptions since there are more than a single option

Reply:
fixed